### PR TITLE
font-sketchybar-app-font 2.0.28 (new cask)

### PR DESCRIPTION
**Pull request by chrede88**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

The `brew install --cask` with `HOMEBREW_NO_INSTALL_FROM_API=1` didn't work for me. This is my first PR to `brew`, so I'm not sure exactly what I could've done wrong. Please advise me on what I need to do in order for this to work.

### CODE REVIEW

File: Casks/font/font-s/font-sketchybar-app-font.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "font-sketchybar-app-font" do
+  version "2.0.28"
+  sha256 "e49ee3281aca67634c2e7c0261d898226149664e9842b7fe61af8c4726d1f1de"
+
+  url "https://github.com/kvndrsslr/sketchybar-app-font/releases/download/v#{version}/sketchybar-app-font.ttf"
+  name "sketchybar-app-font"
+  desc "Contains many useful app icons used by Sketchybar"
```
Comment (bevanjkay) [inline Casks/font/font-s/font-sketchybar-app-font.rb:7]:
```suggestion
```

We no longer add a `desc` stanza to fonts.

File: Casks/font/font-s/font-sketchybar-app-font.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  homepage "https://github.com/kvndrsslr/sketchybar-app-font"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
```
Comment (bevanjkay) [inline Casks/font/font-s/font-sketchybar-app-font.rb:14]:
```suggestion
```

The `livecheck` should be able to be omitted here, as the default `git` strategy will be inferred automatically from the url.

**Comment by bevanjkay** [comment]
Thank you for your first contribution to `homebrew-cask` @chrede88 
I have left a couple of comments above.

File: Casks/font/font-s/font-sketchybar-app-font.rb (hunk 1 above)
Comment (chrede88) [reply Casks/font/font-s/font-sketchybar-app-font.rb]: Removed

File: Casks/font/font-s/font-sketchybar-app-font.rb (hunk 2 above)
Comment (chrede88) [reply Casks/font/font-s/font-sketchybar-app-font.rb]: Removed

**Comment by chrede88** [comment]
> Thank you for your first contribution to `homebrew-cask` @chrede88 I have left a couple of comments above.

Thank you for the fast review. I've removed the `desc` and `livecheck` as suggested.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**